src/include/sof/audio/drc/drc.h

```diff
@@ -15,10 +15,12 @@
 struct audio_stream;
 struct comp_dev;
 
-/* DRC_MAX_PRE_DELAY_FRAMES needs to be a 2^N number */
-#define DRC_MAX_PRE_DELAY_FRAMES 1024
-#define DRC_MAX_PRE_DELAY_FRAMES_MASK (DRC_MAX_PRE_DELAY_FRAMES - 1)
-#define DRC_DEFAULT_PRE_DELAY_FRAMES 256
+#ifndef CONFIG_DRC_MAX_PRE_DELAY_FRAMES
```

Comment:
Is this needed since the Kconfig value will be mandatory on all builds and be 512 by default ?

Reply:
I think it would be needed for testbench purpose. e.g. ./scripts/rebuild-testbench.sh

Reply:
ah, good point. we need to think about how test bench can use Kconfig....

Reply:
can you leaver a comment/todo so when we fix it we can remove it? because it does look weird. @marc-hb FYI

Reply:
How about:
```suggestion
#ifdef CONFIG_LIBRARY
#define CONFIG_DRC_MAX_PRE_DELAY_FRAMES 512
#endif
```
This makes sure firmware compilation fails if someone breaks the new Kconfig above. Plus: self-commenting code :-)